requests/__init__.py

```diff
@@ -70,6 +70,11 @@ def check_compatibility(urllib3_version, chardet_version):
     assert minor < 1
     assert patch >= 2
 
+def _check_cryptography(cryptography_version):
+    # cryptography < 1.3.4
+    if list(map(int, cryptography_version.split('.'))) < [1, 3, 4]:
```

Comment:
This check will break if for any reason cryptography has a non-numerical version string. As in other places, we should aim to tolerate that by assuming that weird cryptography version numbers will be fine.